Лаб.работа 2/задача_2.py

```diff
@@ -0,0 +1,16 @@
+salary = 5000  # Ежемесячная зарплата
+spend = 6000  # Траты за первый месяц
+months = 10  # Количество месяцев, которое планируется протянуть без долгов
+increase = 0.03  # Ежемесячный рост цен
+
+# Расчитываем подушку безопасности, чтобы протянуть 10 месяцев без долгов
+money_capital = 0
+current_spend = spend
+for _ in range(months):
+  money_capital += current_spend
+  current_spend *= (1 + increase)
```

Comment:
Хорошая работа!
Но эти две строки сделаны не по PEP8. Не хватает еще 2 пробелов в начале.
```suggestion
    money_capital += current_spend
    current_spend *= (1 + increase)
```